routers/stream_router.py

```
@@ -1,11 +1,25 @@
+@router.post("/emg", response_model=StreamAckResponseSchema)
+def receive_stream(request: StreamRequestSchema):
+    try:
+        request.validate_window_matches_samples()
+
+        return StreamAckResponseSchema(
+            status="success",
+            message="EMG stream received successfully"
+        )
+
+    except ValueError as e:
+        raise HTTPException(status_code=400, detail=str(e))
```

Ruff (0.15.9)

[warning] Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)